lib/tasks/webpacker/verify.rake

```diff
@@ -1,17 +0,0 @@
-require "webpacker/configuration"
-
-namespace :webpacker do
-  namespace :install do
-    desc "Verifies if webpacker is installed"
-    task :verify do
-      begin
-        File.read(Webpacker::Configuration.file_path)
```

Comment:
Is it necessary? The whole purpose is to check if the config file can be read on the given path (we could have used `File.exists?(path))`, but I guess this works with begin and rescue.